charts/kong/templates/deployment.yaml

```diff
@@ -29,6 +29,8 @@ spec:
         {{- if .Values.podAnnotations }}
 {{ toYaml .Values.podAnnotations | indent 8 }}
         {{- end }}
+        kuma.io/gateway: enabled
+        traffic.sidecar.istio.io/includeInboundPorts: ""
```

Comment:
This is same as the previous commit. The annotations are still at pod level instead of Deployment level. Deployment annotations will go under Line 7 (in a new annotations block).

Reply:
I would like to replicate the same behavior as https://github.com/Kong/kubernetes-ingress-controller/pull/564